modules/custom/dkan_common/src/Tests/Mock/Chain.php

```diff
@@ -42,6 +42,13 @@ public function add($objectClass, $method, $return, $storeId = NULL) {
     return $this;
   }
 
+  /**
+   * Get stored input.
+   */
+  public function getStoredInput($id) {
+    return (isset($this->store[$id])) ? $this->store[$id] : NULL;
```

Comment:
The parens around `isset` aren't necessary, but furthermore, in PHP 7, the null coalescing operator can shorten the line to just:

`return $this->store[$id] ?? NULL;`